src/TSHelper.ts

```diff
@@ -659,11 +659,12 @@ export function getFunctionContextType(type: ts.Type, checker: ts.TypeChecker):
     return reduceContextTypes(signatureDeclarations.map(s => getDeclarationContextType(s, checker)));
 }
 
-export function escapeString(text: string): string {
+export function escapeString(text: string, exclude: RegExp[] = []): string {
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String
     const escapeSequences: Array<[RegExp, string]> = [
         [/[\\]/g, "\\\\"],
-        [/[\']/g, "\\'"],
+        [/\\'/g, "\\'"],
```

Comment:
This replacement does nothing.
I guess it can be safely removed, it wasn't ever required anyway.